pipelines/LSSTCam/quickLook.yaml

```diff
@@ -8,6 +8,9 @@ imports:
       - fgcmBuildFromIsolatedStars
       - fgcmFitCycle
       - fgcmOutputProducts
+      # We exclude calibrateImage here in order to restore the performance-
+      # oriented task default configs.
```

Comment:
I was confused about this comment.  You mean we do not want to import the DRP.yaml calibrateImage config overrides?

Reply:
Right; we're going back to the task defaults, which are effectively the AP `calibrateImage` configs, since AP's goals are much more aligned with quickLook's than DRP's are.  I'll reword the comment to clarify that.